src/coreComponents/finiteElement/elementFormulations/FiniteElementBase.hpp

```diff
@@ -79,18 +82,117 @@ class FiniteElementBase
    */
   virtual ~FiniteElementBase() = default;
 
+  /**
+   * @struct StackVariables
+   * @brief Kernel variables allocated on the stack.
+   *
+   * Contains variables that will be allocated on the stack. Used only by Virtual Element classes to
+   * hold the computed projections of basis functions
+   */
+  struct StackVariables
+  {
+    /**
+     * Default constructor
+     */
+    GEOSX_HOST_DEVICE
+    StackVariables()
+    {}
+  };
+
+  /**
+   * @struct Initialization
+   * @brief Variables used to initialize the class.
+   */
+  struct Initialization
+  {
+    /**
+     * Constructor
+     */
+    GEOSX_HOST_DEVICE
+    Initialization()
+    {}
+  };
```

Comment:
Given what you are going to store in this `struct` (geometry and topology of all the cells) for VEM, something like `MeshData` or `MeshInfo` would be a more descriptive name. And then you would keep the `initialize` for the function that fills this mesh data. Or do you think this `struct` could later also be used for non-mesh data in the future?